project/sendgrid_backend.py

```diff
@@ -14,16 +15,6 @@
 )
 from sgbackend import SendGridBackend
 
-try:
-    from urllib.error import HTTPError  # pragma: no cover
-except ImportError:  # pragma: no cover
-    from urllib2 import HTTPError  # pragma: no cover
-
-try:
-    import rfc822
-except ImportError:
-    import email.utils as rfc822
-
 
```

Comment:
@okaycj did you remove this or was it removed by the linter? If it's the latter case - are we ok with this?

Reply:
I removed it.  I am okay with this as these types of imports are to support varying versions of python.  Especially from python2 to python3.

Reply:
Updated import in commit https://github.com/lookit/lookit-api/pull/1397/commits/e59417ad2913041224a550a720ad6ee83dc3d3bb.